Sources/TuistAcceptanceTesting/TuistAcceptanceTestCase+Extra.swift

```diff
@@ -78,4 +78,58 @@ extension TuistAcceptanceTestCase {
             return
         }
     }
+
+    /// Asserts that a simulated location has been added to a specific testable target.
+    /// - Parameters:
+    ///   - xcodeprojPath: A path to specify `.xcodeproj` file if it is in a different path.
+    ///   - scheme: A specific scheme name.
+    ///   - testTarget: A specific test target name.
+    ///   - simulatedLocation: A simulated location. This value can be passed a `location string` or a `GPX filename`.
+    ///   For example, "Rio de Janeiro, Brazil" or "Grand Canyon.gpx".
+    public func XCTAssertSimulatedLocationAdded(
```

Comment:
nit: Can we rename this to `XCTAssertContainsSimulatedLocation`? It's more inline with how we name these custom assert methods.

Reply:
Sure! That makes sense!